src/Model/Package.php

```diff
@@ -46,12 +50,22 @@ public function getSummary()
     }
 
     /**
-     * requireDefaultRecords() gets abused to update the information on dev/build.
+     * Queue up a job to check for updates to packages if there isn't a pending job in the queue already
      */
     public function requireDefaultRecords()
     {
         parent::requireDefaultRecords();
-        $task = Injector::inst()->create(UpdatePackageInfoTask::class);
-        $task->run(null);
+
+        $pendingJobs = QueuedJobDescriptor::get()->filter([
+            'Implementation' => CheckForUpdatesJob::class,
+            'JobStatus' => QueuedJob::STATUS_NEW
```

Comment:
Shouldn't you include jobs that might be initialising/running? `QueuedJob::STATUS_(INIT/RUNNING)`